CODIGO/Declares.bas

```diff
@@ -163,6 +172,7 @@ Public FxLoops               As Long
 Public mFadingStatus         As Byte
 Public mFadingMusicMod       As Long
 Public NextMP3               As Byte
+Public CdTimes(e_CdTypes.CDCount) As Long
```

Comment:
mmm CdTimes is not a great name.

Would suggest renaming this to `CoolDownTimestamp` o `CooldownLastTick` or something more self explanatory.

optional

Reply:
Yeah CDLastTick could be better but lets keep it as it is since we we need to change it in many places, since we are using the same name in server, resources and the index generation tool
"cd" is common terminology for cooldown so I think it fine to use it

Reply:
Okay